third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -1370,6 +1371,146 @@ struct RsqrtOpConversion
   bool ftz;
 };
 
+static inline std::pair<Value, Value>
+ScaleUpIfDenorm(ConversionPatternRewriter &rewriter, Location loc,
```

Comment:
Nit: please following naming convention; should be `scaleUp..`. similarly for the following `scaleDown..`